language-plutus-core/test/types/recursiveUnwrap.plc

```diff
@@ -1,18 +0,0 @@
-(program 1.0.0
```

Comment:
This test we discussed how to fix in your previous PR and you even did it there.